Mathlib/Algebra/IndicatorFunction.lean

```diff
@@ -306,9 +306,8 @@ theorem mulIndicator_one_preimage (s : Set M) :
 
 @[to_additive]
 theorem mulIndicator_const_preimage_eq_union (U : Set α) (s : Set M) (a : M) [Decidable (a ∈ s)]
-    [Decidable ((1 : M) ∈ s)] :
-    (U.mulIndicator fun _ => a) ⁻¹' s = (if a ∈ s then U else ∅) ∪ if (1 : M) ∈ s then Uᶜ else ∅ :=
-  by
+    [Decidable ((1 : M) ∈ s)] : (U.mulIndicator fun _ => a) ⁻¹' s =
+    (if a ∈ s then U else ∅) ∪ if (1 : M) ∈ s then Uᶜ else ∅ := by
```

Comment:
```suggestion
    [Decidable ((1 : M) ∈ s)] : (U.mulIndicator fun _ => a) ⁻¹' s =
      (if a ∈ s then U else ∅) ∪ if (1 : M) ∈ s then Uᶜ else ∅ := by
```
Not a fan of the original indentation pattern but I'm not sure what would be best. The return type is a "group" in the sense that hanging indents for large return types should not be at the same level as the surrounding group for the binders and return type.

Reply:
I'm going to let you suggest your changes and then I'll batch them in one commit.